### DISCUSSION TITLE
Welcome to OpenUpgrade Discussions!

### DISCUSSION THREAD

**Discussion by pedrobaeza** [OCA/OpenUpgrade#2775] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mileo** [comment]
Nice!!!

Could some one enable it on https://github.com/OCA/l10n-brazil too.

**Reply by pedrobaeza** [reply]
It should be done by a psc of such repo.
